JOSESwift/Sources/CryptoImplementation/AES.swift

```diff
@@ -29,15 +29,18 @@ internal enum AESError: Error {
     case encryptingFailed(description: String)
     case decryptingFailed(description: String)
     case cannotPerformOperationOnEmptyDataBuffer
+    case invalidAlgorithm
 }
 
 fileprivate extension ContentEncryptionAlgorithm {
     var ccAlgorithm: CCAlgorithm {
         switch self {
         case .A256CBCHS512:
+            // 128 referes to the block size in this case.
             return CCAlgorithm(kCCAlgorithmAES128)
```

Comment:
As opposed to key, I suppose?

Reply:
Yes exactly! i checked and `kCCAlgorithmAES128` is actually deprecated in favor of just `kCCAlgorithmAES` due to this exact ambiguity. I updated it to `kCCAlgorithmAES`.